syntax = "proto2";

package reviewers;

message Projection {
  optional string version = 1;
  repeated ProjectionEntry entries = 2;
}

message ProjectionEntry {
  optional string author = 1;
  repeated float rating = 2;
}